### PR TITLE
Responsive layout for logos page

### PR DESCRIPTION
**Pull request by kennethlarsen**

### CODE REVIEW

**Comment by locks** [comment]
@kennethlarsen can you include some screenshots/gifs, please? :]

**Comment by kennethlarsen** [comment]
@locks Here you go!

ember_screen

ember

File: source/stylesheets/site.css.scss [hunk 1]
```diff
@@ -17,6 +17,7 @@
 @import 'components/legal';
 @import 'components/security';
 @import 'components/sponsors';
+@import 'components/logos.css.scss';
```
Comment (acorncom) [inline source/stylesheets/site.css.scss]: We don't actually need the extensions here

**Comment by acorncom** [comment]
Nice work! One small tweak and then we'll merge this

**Comment by acorncom** [comment]
Thanks!

**Pull request merged**